plugins/pulp_rpm/plugins/importers/yum/repomd/packages.py

```diff
@@ -48,7 +48,7 @@ def package_list_generator(xml_handle, package_tag, process_func=None):
     # I know. This is a terrible misuse of SyntaxError. Don't blame the messenger.
     except SyntaxError:
         _LOGGER.error('failed to parse XML metadata file')
-        return
+        raise
```

Comment:
Yes, this change is intentional so that the Syntax error can be properly traced from the log rather than being replaced by a less specific error later on.